src/Pages/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import './LandingPage.css';
+import Header from '../../Shared/Header/Header';
+import Corousel from '../../Shared/Corousel/Corousel';
+import Banner from '../../Shared/Banner/Banner';
+import ProductCard from '../../Shared/ProductCard/ProductCard';
+import Footer from '../../Shared/Footer/Footer';
+import {categories,topCategories, featuredProducts, testimonials} from '../../Data';
+import blitheLogo from '../../Assets/Images/Blithe-logo.png';
+
+const LandingPage = () =>{
+
+    console.log("testimonials", testimonials);
+
+    return(
+        <>
+            <Header/>
+
+            {/* <!-- top categories --> */}
+            <div className="col-12 category-container">
+            {
+                topCategories.length?
+                    topCategories.map(ctgry=>
+                        <button key={ctgry.id} className="button category-item">
+                        <img className="category sm"
+                            src={ctgry.img}
+                            alt={ctgry.title} />
+                        <p className='text-sm'>{ctgry.title}</p>
```

Comment:
Suggestion: naming should be improve like: ctgry can be name as `category`. Naming should make some sense of what is represent.

Reply:
done!